### ISSUE TITLE
Create combined k8s manifest with all objects

### ISSUE THREAD

**Issue by rasheedamir** [stakater/IngressMonitorController#87]
To simplify the installation of IMC there should be one k8s manifests which includes all objects i.e. rbac, configmap and deployment

And then add in README how the user can just use that one manifest to install IMC

**Comment by rasheedamir** [comment]
Step 1: Deploy imc to your cluster with command:

```
kubectl apply -f https://raw.githubusercontent.com/stakater/IngressMonitorController/master/deployments/kubernetes/manifests/imc.yaml
```

Step 2: Configure the configmap

```
provide command for it ???
```

**Comment by ahmadiq** [comment]
@rasheedamir is there a way to update specific properties of the configmap. i haven't found one. Also overall it may be better to have the yaml file downloaded first since we are going to have a unified yaml. The namespaces may need to be edited, and will also be useful if needed for deletion/re-creation.

**Comment by rasheedamir** [comment]
Yeah its fine and we can mention that download this unified file; update particular stuff and then apply it. The whole idea is to make it really simple for the user to give it a try and get it working.

**Comment by ahmadiq** [comment]
I've refined the text a bit for this.
